_posts/2021-03-23-connector-build.md

```diff
@@ -0,0 +1,240 @@
+```
+
+You can also double check the status of KafkaConnect, which will now contains all connectors which you added:
+
+```yaml
+...
+  status:
+    conditions:
+    - lastTransitionTime: "2021-03-23T16:31:11.204228Z"
+      status: "True"
+      type: Ready
+    connectorPlugins:
+    - class: cz.scholz.kafka.connect.echosink.EchoSinkConnector
+      type: sink
+      version: 1.0.0
+    - class: io.debezium.connector.postgresql.PostgresConnector
+      type: source
+      version: 1.4.2.Final
+    - class: org.apache.camel.kafkaconnector.CamelSinkConnector
+      type: sink
+      version: 0.8.0
+    - class: org.apache.camel.kafkaconnector.CamelSourceConnector
+      type: source
+      version: 0.8.0
+    - class: org.apache.camel.kafkaconnector.timer.CamelTimerSourceConnector
+      type: source
+      version: 0.8.0
+    - class: org.apache.kafka.connect.file.FileStreamSinkConnector
+      type: sink
+      version: 2.7.0
+    - class: org.apache.kafka.connect.file.FileStreamSourceConnector
+      type: source
+      version: 2.7.0
+    - class: org.apache.kafka.connect.mirror.MirrorCheckpointConnector
+      type: source
+      version: "1"
+    - class: org.apache.kafka.connect.mirror.MirrorHeartbeatConnector
+      type: source
+      version: "1"
+    - class: org.apache.kafka.connect.mirror.MirrorSourceConnector
+      type: source
+      version: "1"
+    labelSelector: strimzi.io/cluster=my-connect-cluster,strimzi.io/name=my-connect-cluster-connect,strimzi.io/kind=KafkaConnect
+    observedGeneration: 1
+    replicas: 1
+    url: http://my-connect-cluster-connect-api.kafka.svc:8083
+```
+
+Now you can create KafkaConnector resource and use there any of the connectors which you added to the build section.
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnector
+metadata:
+  name: telegram-connector
+  labels:
+    strimzi.io/cluster: my-connect-cluster
+spec:
+  class: org.apache.camel.kafkaconnector.timer.CamelTimerSourceConnector
+  tasksMax: 5
+  config:
+    camel.source.path.timerName: timer
+    topics: timer-topic
+    value.converter.schemas.enable: false
+    transforms: HoistField,InsertField,ReplaceField
+    transforms.HoistField.type: org.apache.kafka.connect.transforms.HoistField$Value
+    transforms.HoistField.field: originalValue
+    transforms.InsertField.type: org.apache.kafka.connect.transforms.InsertField$Value
+    transforms.InsertField.timestamp.field: timestamp
+    transforms.InsertField.static.field: message
+    transforms.InsertField.static.value: 'Hello World'
+    transforms.ReplaceField.type: org.apache.kafka.connect.transforms.ReplaceField$Value
+    transforms.ReplaceField.blacklist: originalValue
+```
+
+You can verify that connector is working by attaching kafka client to a connector topic:
+```bash
+kubectl -n kafka exec my-cluster-kafka-0 -c kafka -i -t -- bin/kafka-console-consumer.sh  --bootstrap-server localhost:9092 --topic timer-topic
```

Comment:
Please use `kubectl run` to start a new pod (see our quickstarts where you can copy the command) instead of using `kubectl exec` into a broker pod. It is a bad practice.